ultralytics_yolov5.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: YOLOv5
+summary: YOLOv5 in PyTorch > ONNX > CoreML > TFLite
+image: ultralytics_yolov5_img0.jpg
+author: Ultralytics LLC
+tags: [vision, scriptable]
+github-link: https://github.com/ultralytics/yolov5
+github-id: ultralytics/yolov5
+featured_image_1: ultralytics_yolov5_img1.jpg
+featured_image_2: ultralytics_yolov5_img2.png
+accelerator: cuda-optional
+---
+
+🚀  This guide explains how to load YOLOv5 from PyTorch Hub [https://pytorch.org/hub/](https://pytorch.org/hub/)
+
+## Before You Start
+
+Start from a working python environment with **Python>=3.8** and **PyTorch>=1.6** installed, as well as `pyyaml>=5.3` for reading YOLOv5 configuration files. To install PyTorch see [https://pytorch.org/get-started/locally/](https://pytorch.org/get-started/locally/). To install dependencies:
+```bash
+$ pip install -U opencv-python pillow pyyaml tqdm  # install dependencies
+```
+
+Note that cloning the [https://github.com/ultralytics/yolov5](https://github.com/ultralytics/yolov5) repository is **not** required.
```

Comment:
remove this line. it's redundant for pytorch hub models

Reply:
Done.